lawnchair/src/app/lawnchair/LawnchairLauncher.kt

```
@@ -152,21 +162,106 @@ class LawnchairLauncher : QuickstepLauncher() {
+            if (toState is AllAppsState) {
+                // Self-heals a cold-start race: the paged grid's first refresh can run before
+                // the model finishes loading apps (most noticeable right after a fresh
+                // install), leaving it empty until something else happens to refresh it. This
+                // is a cheap no-op otherwise (also no-ops when paged mode isn't active).
+                appsView.refreshPagedGridView()
+                if (!preferenceManager2.rememberPosition.firstCached()) {
+                    mAppsView?.findViewById<AllAppsPagedGridView>(R.id.apps_paged_grid_view)?.setCurrentPage(0)
+                }
+            }
+        }
+        override fun onStateTransitionComplete(finalState: LauncherState) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)